### ISSUE TITLE
Missing the 'notifications' scope

### ISSUE THREAD

**Issue by pezholio** [andrew/octobox#42]
Just tried to set up Octobox following the instructions, and after logging in, I see a somewhat bare dashboard:

![download 12](https://cloud.githubusercontent.com/assets/109774/21312168/a4dc52aa-c5e2-11e6-8a58-e9b867a939ec.png)

I then click 'refresh' and see this error:

```
GET https://api.github.com/notifications?all=true&per_page=100: 403 - Missing the 'notifications' scope. // See: https://developer.github.com/v3
```

**Comment by m1guelpf** [comment]
Fixed in #43

**Comment by andrew** [comment]
@pezholio just merged https://github.com/andrew/octobox/pull/43, pull and re-login 👍
